### ISSUE TITLE
Suggestion /pokebattle update

### ISSUE THREAD

**Issue by ohhnate** [Dysth/Generations#315]
- Allow /pokebattle to accept <player> and <NPC Trainer> as the two parameters. Or just make /pokebattle3 that allows a 1v1 battle between player and chosen NPC Trainer.

**Comment by Azazel-mp** [comment]
I agree, an updated /pokebattle would be fantastic. For a custom map I wanted to force a battle with a golden pokemon, but couldn't put <pokemon> or the pokemon specs as arguments.
